src/Digdir.Domain.Dialogporten.Infrastructure/Altinn/Authorization/AltinnAuthorizationClient.cs

```diff
@@ -140,9 +142,22 @@ private async Task<DialogDetailsAuthorizationResult> PerformDialogDetailsAuthori
     {
         var xacmlJsonRequest = DecisionRequestHelper.CreateDialogDetailsRequest(request);
         var xamlJsonResponse = await SendPdpRequest(xacmlJsonRequest, cancellationToken);
+        LogIfIndeterminate(xamlJsonResponse, xacmlJsonRequest);
+
         return DecisionRequestHelper.CreateDialogDetailsResponse(request.AltinnActions, xamlJsonResponse);
     }
 
+    private void LogIfIndeterminate(XacmlJsonResponse? response, XacmlJsonRequestRoot request)
+    {
+        if (response?.Response != null && response.Response.Any(result => result.Decision == "Indeterminate"))
+        {
+            _logger.LogError(
+                "Authorization request to {Url} returned decision Indeterminate. Request: {RequestJson}",
+                AuthorizeUrl, JsonSerializer.Serialize(request, SerializerOptions)
+            );
```

Comment:
```suggestion
            _logger.LogError(
                "Authorization request to {Url} returned decision Indeterminate. Request: {@RequestJson}",
                AuthorizeUrl, request
            );
```

Serilog støtter serialisering av objekter ved bruk [@ operatoren](https://github.com/serilog/serilog/wiki/Structured-Data#preserving-object-structure). Et eksempel jeg gjorde lokalt: 
![image](https://github.com/user-attachments/assets/900f3bd2-ae72-425b-a4b5-a296e43eb6f7)
Så dersom du ikke har eksplisitt behov for å serialisere det på en spesifikk måte (SerializerOptions), så kan du simplifisere dette.

Reply:
hm, vil vi ha disse dataene i loggene? 🤔 Er det noe data som er sensitiv i den requesten?

Reply:
Uten XACML-requesten kan det bli vanskelig å skulle reprodusere feilen. Men vi er neppe avhengig av å vite hvilken person det er, så vi bør se på å bare gjøre en search/replace av `Value` der `AttributeId` er `urn:altinn:person:identifier-no`

Reply:
Oppdatert med fjerning av pid før logging, samt forenklet serialisering.